packages/plugins/DecentralizedSearch/src/SNSAdaptor/SearchResultInspector.tsx

```diff
@@ -0,0 +1,152 @@
+import { useRef, useEffect, useState, useMemo } from 'react'
+import { Hidden } from '@masknet/shared'
+import { first } from 'lodash-es'
+import { TabContext } from '@mui/lab'
+import { useAvailablePlugins, getProfileTabContent } from '@masknet/plugin-infra'
+import {
+    useActivatedPluginSNSAdaptor,
+    useActivatedPluginsSNSAdaptor,
+    usePluginI18NField,
+} from '@masknet/plugin-infra/content-script'
+import { Tab } from '@mui/material'
+import { useWeb3State } from '@masknet/web3-hooks-base'
+import { useAddressTypeBaseOnChainIdList } from '@masknet/web3-hooks-evm'
+import { ChainId, AddressType } from '@masknet/web3-shared-evm'
+import { makeStyles, MaskTabList, useTabs } from '@masknet/theme'
+import { SocialAddressType, resolveSearchKeywordType } from '@masknet/web3-shared-base'
+import { DecentralizedSearchPostExtraInfoWrapper } from './DecentralizedSearchPostExtraInfoWrapper.js'
+import { PluginID, EMPTY_LIST, NetworkPluginID } from '@masknet/shared-base'
+import { LoadingContent } from './LoadingContent.js'
+import { EmptyContent } from './EmptyContent.js'
+import { LoadFailedContent } from './LoadFailedContent.js'
+
+const useStyles = makeStyles()((theme) => ({
+    tabs: {
+        display: 'flex',
+        position: 'relative',
+        padding: theme.spacing(0, 2),
+    },
+    content: {
+        position: 'relative',
+        background: theme.palette.common.white,
+        maxHeight: 478,
+        overflow: 'scroll',
+    },
+}))
+
+export function SearchResultInspector(props: { keyword: string }) {
+    const ENS_Plugin = useActivatedPluginSNSAdaptor(PluginID.ENS, 'any')
```

Comment:
Don't forget to add `DecentralizedSearch` to `PluginID`.

Reply:
We need to use ENS plugin in the DecentralizedSearch.